### ISSUE TITLE
False-positive when adding optional parameter to constructor

### ISSUE THREAD

**Issue by vjik** [Roave/BackwardCompatibilityCheck#978]
Example: https://github.com/yiisoft/log/pull/128

Before:

```php
public function __construct()
{
    // ...
}
```

After:

```php
public function __construct(array $levels = [])
{
    // ...
}
```

Result:

```
[BC] ADDED: Parameter levels was added to Method __construct() of class Yiisoft\Log\Target
```
